store/storetest/channel_store.go

```diff
@@ -24,6 +24,7 @@ import (
 
 type SqlSupplier interface {
 	GetMaster() *gorp.DbMap
+	DriverName() string
```

Comment:
why is this change in this PR?  (same for the changes below)

Reply:
I changed the version of mysql to 5.6, and I've being debugging this failing tests during hours and hours, and only fails on certain circustances on MySQL 5.6, I haven't find the way to reproduce that and I don't want to invest more time on this before this get merged. We can iterate later, but I've given up with this specific test case.